bottles/backend/wine/winecommand.py

```diff
@@ -261,7 +261,8 @@ def get_env(self, environment: dict = None, return_steam_env: bool = False, retu
             for lib in gst_libs:
                 if os.path.exists(os.path.join(runner_path, lib)):
                     gst_env_path.append(os.path.join(runner_path, lib))
-            env.add("GST_PLUGIN_SYSTEM_PATH", ":".join(gst_env_path), override=True)
+            if len(gst_env_path) > 0:
```

Comment:
can you add a comment here that reference the PR?

Reply:
It was PR #2258

Reply:
I mean, adding a comment to the source to reference this PR, for future developers

Reply:
Why can't one use `git blame`?

Reply:
I always forgot about it